ios/RCTCameraManager.m

```diff
@@ -918,6 +918,29 @@ - (NSData *) nsDataFromSampleBuffer:(CMSampleBufferRef)sampleBuffer
 //    //NSLog(@"Video Output: dropped an buffer");
 //}
 
+- (CGImageRef) resizeImage:(CGImageRef)image
+                   maxSize:(int)size
+{
+    float width = CGImageGetWidth(image);
+    float height = CGImageGetHeight(image);
+    float scale = size / MAX(width, height);
```

Comment:
If width and height are smaller than size, it will upscale the image. I don't think that's desirable.

Reply:
Now if scale >= 1 we return a copy of the original image, so we don't upscale it anymore.

Reply:
We return a copy so we can do some manipulation later on if needed and we can release it from memory also.